components/block/hero.vue

```diff
@@ -12,30 +12,26 @@
               <span class="block xl:inline">{{ t('hero.title') }}</span>
             </h1>
             <p
-              class="mt-5 max-w-[302px] mx-auto font-nunito text-subtitle text-zinc-700  sm:text-headline-5 sm:max-w-[652px]"
+              class="mt-5 max-w-xs mx-auto font-nunito text-subtitle text-zinc-700  sm:text-headline-5 sm:max-w-2xl"
             >
               {{ t('hero.description') }}
             </p>
           </div>
           <div
-            class="mt-8 gap-2 mx-auto flex flex-col items-center justify-center h-24 sm:mt-12 sm:max-w-sm sm:h-16 sm:flex-row sm:gap-4"
+            class="mt-8 gap-2 mx-auto flex flex-col items-center justify-center h-24 sm:h-12 sm:mt-12 sm:max-w-sm sm:flex-row sm:gap-4"
```

Comment:
can you check the desktop view button on the Our Team section, please, so they look the same?